tsconfig.json

```diff
@@ -9,15 +9,10 @@
     "outDir": "build",
     "jsx": "react-jsx",
     "strict": true,
-    "target": "es2021",
-    "types": [], //reconsider this later
```

Comment:
If I read the docs correctly, this field is mostly for explicitly adding type files/ path to type definitions. We are not really doing this, so we can remove this configuration.

See https://www.typescriptlang.org/tsconfig/#types